components/layouts/DashboardLayout.tsx

```diff
@@ -14,7 +14,7 @@ export default function DashboardLayout({ children }: DashboardLayoutProps) {
             <main
             className={`flex min-h-screen flex-col items-center dark:bg-slate-900 ${inter.className}`}
             >
-                <div>
+                <div className="w-full">
```

Comment:
Perhaps the size of the width may be capped to let's say `max-w-7xl`, as otherwise, the cards will look too stretched out.

Without size cap:
![image](https://github.com/projectcatena/cloudlabs-app/assets/63692048/fc9cc3d0-27b8-4760-98fa-5a0c9a1be26a)

With size cap:
![image](https://github.com/projectcatena/cloudlabs-app/assets/63692048/43135289-11a0-4c70-b7c3-e17d03ff47e1)